pkg/network/protocols/http/monitor.go

```diff
@@ -117,6 +117,8 @@ func NewMonitor(c *config.Config, offsets []manager.ConstantEditor, sockFD *ebpf
 	statkeeper := newHTTPStatkeeper(c, telemetry)
 	processMonitor := monitor.GetProcessMonitor()
 
+	state = Running
```

Comment:
let's do that in a different PR

Reply:
I didn't add it, it's your code from main